README.md

```diff
@@ -94,6 +94,24 @@ documentation](https://doc.zebra.zfnd.org) for the external API of our crates,
 as well as [internal documentation](https://doc-internal.zebra.zfnd.org) for
 private APIs.
 
+## Alpha Release Details
```

Comment:
Can we move these sections to the top of the readme, so people see them first?

And can we link to the alpha release tag?

Reply:
did we already decide on what the tag will be?

Reply:
Also, I'm thinking it might be a good idea to link to the `CHANGELOG.md` and have the alpha tag link be in the changelog, which afaik is pretty standard.

Reply:
> did we already decide on what the tag will be?
> 
> If i have to pick one I'd pick `v1.0.0-alpha.0`

We're currently at `zebrad 3.0.0-alpha.0`.

Reply:
Actually we decided to go with 1.0.0-alpha